### PR TITLE
fix wstring path conversions

### PR DESCRIPTION
**Pull request by despair86**
aw crap i lost the other half of this where i started doing the refactoring in #1251

### CODE REVIEW

**Comment by despair86** [comment]
this one actually fixes the windows build altogether

**Comment by majestrate** [comment]
this ready?

File: llarp/context.cpp [hunk 1]
```diff
@@ -50,7 +50,7 @@ namespace llarp
       jobQueueSize = 1024;
     logic = std::make_shared<Logic>(jobQueueSize);
 
-    nodedb_dir = config->router.m_dataDir / nodedb_dirname;
+    nodedb_dir = fs::path(config->router.m_dataDir / nodedb_dirname).string();
```
Comment (majestrate) [inline llarp/context.cpp:53]: nodedb_dir can probably also be an `fs::path`

Comment (despair86) [reply llarp/context.cpp:53]: k will do

**Comment by despair86** [comment]
ok this is ready @majestrate

**Comment by majestrate** [comment]
@despair86 revert last commit, i think it was fine before that.

**Comment by despair86** [comment]
done

File: llarp/util/logging/logger_internal.cpp [hunk 2]
```diff
@@ -9,6 +9,6 @@ namespace llarp
   {
     std::chrono::time_point<std::chrono::system_clock, std::chrono::milliseconds> now{
         std::chrono::milliseconds{ts.now}};
-    return date::operator<<(out, now) << " UTC [+" << ts.delta << "]";
+    return date::operator<<(out, now) << " GMT [+" << ts.delta << "]";
```
Comment (jagerman) [inline llarp/util/logging/logger_internal.cpp:12]: Why this change?

Comment (notlesh) [reply llarp/util/logging/logger_internal.cpp:12]: :thinking:
